api/app/controllers/api/v1/pending_entitlements_controller.rb

```diff
@@ -13,7 +13,7 @@ class PendingEntitlementsController < ApplicationController
       INCLUDES = %i[entitlement user subject].freeze
 
       resourceful! PendingEntitlement, authorize_options: { except: [:index] } do
-        PendingEntitlement.preload(*INCLUSIONS).in_default_order.filtered(
+        PendingEntitlement.preload(*INCLUSIONS).by_state(:pending).filtered(
```

Comment:
Just flagging that it looks like I caused tests to fail by doing this. I (think I) needed to remove `in_default_order` for the order filter to work. I added the state scope because the default in strong params never gets used; there's always a filters param from the client. But I can also always send `state: pending` from the client.